File: health_project/lib/models/account_dto.dart
class AccountDTO {
  final String username;
  final String password;
  final String phone;
  final bool isLoginPhone;

  const AccountDTO({
    required this.username,
    required this.password,
    required this.phone,
    required this.isLoginPhone,
  });

  factory AccountDTO.fromJson(Map<String, dynamic> json) {
    return AccountDTO(
      username: json['username'],
      password: json['password'],
      phone: json['phone'],
      isLoginPhone: json['isLoginPhone'],
    );
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['username'] = this.username;
    data['password'] = this.password;
    data['phone'] = this.phone;
    data['isLoginPhone'] = this.isLoginPhone;
    return data;
  }
}

File: healt_project_server/src/main/java/com/dh/health/dto/AccountDTO.java
package com.dh.health.dto;

import java.io.Serializable;

import com.fasterxml.jackson.annotation.JsonProperty;

public class AccountDTO implements Serializable{
	
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	
	private String username;
	private String password;
	private String phone;
	@JsonProperty
	private boolean isLoginPhone;
	public AccountDTO() {
		super();
	}
	public AccountDTO(String username, String password, String phone, boolean isLoginPhone) {
		super();
		this.username = username;
		this.password = password;
		this.phone = phone;
		this.isLoginPhone = isLoginPhone;
	}
	public String getUsername() {
		return username;
	}
	public void setUsername(String username) {
		this.username = username;
	}
	public String getPassword() {
		return password;
	}
	public void setPassword(String password) {
		this.password = password;
	}
	public String getPhone() {
		return phone;
	}
	public void setPhone(String phone) {
		this.phone = phone;
	}
	public boolean isLoginPhone() {
		return isLoginPhone;
	}
	public void setPhoneLogin(boolean isLoginPhone) {
		this.isLoginPhone = isLoginPhone;
	}
	
	
}
